code/game/turfs/open/_open.dm

```diff
@@ -298,15 +298,24 @@
 /// Very similar to build_with_rods, this exists to allow consistent behavior between different types in terms of how
 /// Building floors works
 /turf/open/proc/build_with_floor_tiles(obj/item/stack/tile/iron/used_tiles, user)
-	var/obj/structure/lattice/soon_to_be_floor = locate(/obj/structure/lattice, src)
-	if(!soon_to_be_floor)
+	var/obj/structure/lattice/lattice = locate(/obj/structure/lattice, src)
+	if(!has_valid_support() && !lattice)
 		to_chat(user, span_warning("The plating is going to need some support! Place metal rods first."))
```

Comment:
Might be time to turn these into balloon alerts especially since it's more contextual now